doc/apiary/iotagent.apib

```diff
@@ -93,9 +93,9 @@ Mandatory fields are identified in every operation.
 
 
 
-### Retrieve a service [GET]
+### Retrieve a service group [GET]
```

Comment:
The "cannonical name" is "configuration group", isn't it? At least that is the one used along https://thinking-cities.readthedocs.io/en/master/device_api/index.html, as far as I understand.

Reply:
fixed in ad54e4edc563568e1539f201ad40bb3a3154f100